addons/calendar_sms/security/calendar_sms_security.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo noupdate="1">
+    <record id="ir_rule_sms_template_calendar_manager" model="ir.rule">
+        <field name="name">SMS Template: calendar manager CRUD on calendar templates</field>
+        <field name="model_id" ref="sms.model_sms_template"/>
+        <field name="groups" eval="[(4, ref('base.group_partner_manager'))]"/>
+    </record>
+</odoo>
```

Comment:
After retesting the all thing, I remembered why I've done this in the first place. In fact, I need this to be able to create a reminder when the user is a manager if others modules using sms.template were installed. The record rule for calendar_user was just for testing purpose, but not this part.

Reply:
Ok now I see the usage.

However, you need to specify the model_ids to which you give access with:
`<field name="domain_force">[('model_id.model', 'in', ('calendar.event', 'calendar.attendee'))]</field>`
(Just like in event_sms and crm_sms).
Those are the 2 models on which you can select a sms_template_id when creating a calendar.alarm.

Also, it looks like base.group_user has access to alarm.reminder creation, so you should switch from 'base.group_partner_manager' to 'base.group_user' IMO.

(Please re-test this carefully as access rights are touchy).

Reply:
done, I've also created a hidden group in calendar (group_calendar_user). For a reason that I don't understand, without this group and by using directly base.group_user in the record rule, it breaks tests in the module test_mail_full.